deploy/vagrant/pip-req.txt

```diff
@@ -1,3 +1,4 @@
+markupsafe
```

Comment:
markupsafe is a dependency of Jinja2. remove this for now unless we see more problems caused by this missing dependency in Jinja